test-suite/jvm/src/main/scala/org/scalajs/testsuite/utils/Platform.scala

```diff
@@ -34,6 +34,8 @@ object Platform {
 
   def executingInJVMOnLowerThanJDK17: Boolean = jdkVersion < 17
 
+  def executingInJVMOnLowerThanJDK(version: Int): Boolean = jdkVersion < version
```

Comment:
Feel like we should have noticed the simplification potential here earlier :rofl:  Could you follow-up with a PR that removes the hardcoded ones?

FWIW: Also makes me wonder if we should just expose `jdkVersion`? (and make it throw on the JS platform).

Both things can definitely happen later.

Reply:
I'll follow up with a PR for that, yes.